src/components/pokemon-list/pokemon-list.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * Simple component that has no state
+ * and just renders things.
+ */
+export const PokemonList = ({ items }: { items: Pokemon[] }) => {
+  return (
+    <>
```

Comment:
I wrapped the list in a Fragment since React wants all components to return a single element, not a list.